### PR TITLE
(PC-2832) : Script to delete corrupter allocine stocks

### PR DESCRIPTION
**Pull request by DanielBaudry**

### CODE REVIEW

File: tests/scripts/delete_corrupted_allocine_stocks_test.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        PcObject.save(stock)
+
+        # When
+        delete_corrupted_allocine_stocks()
+
+        # Then
+        assert Stock.query.count() == 1
+
+    @clean_database
+    def test_should_not_delete_stock_from_other_provider(self, app):
```
Comment (NerOcrO) [inline tests/scripts/delete_corrupted_allocine_stocks_test.py]:
```suggestion
    def test_should_not_delete_stock_from_other_provider_than_allocine(self, app):
```

File: tests/scripts/delete_corrupted_allocine_stocks_test.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        PcObject.save(stock)
+
+        # When
+        delete_corrupted_allocine_stocks()
+
+        # Then
+        assert Stock.query.count() == 0
+
+    @clean_database
+    def test_should_not_delete_stock_with_new_id_format(self, app):
```
Comment (NerOcrO) [inline tests/scripts/delete_corrupted_allocine_stocks_test.py]:
```suggestion
    def test_should_not_delete_stock_from_allocine_with_new_id_format(self, app):
```

File: tests/scripts/delete_corrupted_allocine_stocks_test.py (hunk 1 above)
Comment (DanielBaudry) [reply tests/scripts/delete_corrupted_allocine_stocks_test.py]: Yes !

File: tests/scripts/delete_corrupted_allocine_stocks_test.py (hunk 2 above)
Comment (DanielBaudry) [reply tests/scripts/delete_corrupted_allocine_stocks_test.py]: Yes

**Pull request merged**